### ISSUE TITLE
"This value is required" errors in 0.10.0

### ISSUE THREAD

**Issue by mafrosis** [jfinkhaeuser/prance#16]
I had to rollback to 0.9.0 as the new release doesn't parse my swagger. The errors look like:

```
ipdb> prance.ResolvingParser(filepath)
*** prance.SwaggerValidationError: 'paths':
    - '/person/alias':
        - 'post':
            - 'parameters':
                - 'referenceObject':
                    - 'required':
                        - '$ref':
                            - 'This value is required'
                - 'parameterObject':
                    - 'required':
                        - 'name':
                            - 'This value is required'
                        - 'in':
                            - 'This value is required'
```

I can attempt to create a small test case for this if necessary, but I won't get to it for a week or so.

**Comment by jfinkhaeuser** [comment]
Huh. Thanks for that. A test case would be useful, yes.

If you want, you can send me the swagger to jens@finkhaeuser.de and I'll extract an anonymized test case. Up to you!

**Comment by x1m** [comment]
Having the same error. Works fine after rolling back to 0.9.0.

**Comment by jfinkhaeuser** [comment]
Yeah, the separate specs also fail with the BaseParser, which does not resolve. That hasn't changed from 0.9, so I don't think that's the only reason. Both validation backends just don't like JSON references that aren't well-formed URLs, and the `petstore-separate` examples have that issue.

So that's something I can *also* fix, and then move forward.

**Comment by jfinkhaeuser** [comment]
Right, figured it out. Turns out to be a small fix, thankfully.

**Comment by jfinkhaeuser** [comment]
Fixed in v0.10.1

**Comment by mafrosis** [comment]
I can confirm this is fixed for me in `v0.11.0` 👍
